interactive-calculator/src/containers/BugetSet/BudgetSet.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
```

Comment:
Interesting using the export default to cut down on having to declare/name the functional component. I like it!